### DISCUSSION TITLE
Conversation Handler for group chats

### DISCUSSION THREAD

**Discussion by TomMago** [python-telegram-bot/python-telegram-bot#3574] [Q&A]
Dear community,

I have a bot with a fairly long conversation handler with many states.
The bot works great for a single user, but we would like to enable it to work in a group chat.

Specifically we want the bot to work in a group chat and all users in the chat should be able to interact with it and change the state like for a single user.

In the documentation it says the ConversationHandler class is not suited for group chats.
Is there any way to make this work?

Suggestions are much appreachiated.

Cheers
Tom

**Comment by Bibo-Joshi** [comment]
https://t.me/pythontelegrambotgroup/662496?thread=662490
